### PR TITLE
Django messages in angular

### PR DESCRIPTION
**Pull request by magul**
__Description:__

As we're moving from Django to Angular we have lost some messages, that were rendered on Django views, but not on Angular component.

Using new api endpoint (`/api/messages/`) and Angular component that's outside `router-outlet` we can render missing messages from Django on Angular views.

__New imports / dependencies:__

* N/A

__Unit Tests:__

* N/A

__What tests do I need to run to validate this change:__

* Register new user and see that it render two messages on Angular homepage regarding this action
* Try to edit organization that's not yours (but being logged in)
* ... and lot more

### CODE REVIEW

**Comment by magul** [comment]
@CorneeSean @juliaanholcer I have to make another pull request, as Github staff have mistakenly detach my fork from CodeForPoznan (see: #856)

I resolved your requests from #852.

File: frontend/src/app/messages/messages.component.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+@Component({
+  selector: 'volontulo-messages',
+  templateUrl: './messages.component.html',
+})
+export class MessagesComponent {
+  messages$: Observable<Message[]>;
+  showMessages = true;
+
+  constructor(private http: Http) {
+    setTimeout(() => this.showMessages = false, 25000);
```
Comment (CorneeSean) [inline frontend/src/app/messages/messages.component.ts]: I guess setTimeout could be initiated when the observable is resolved, not in component constructor directly.

File: frontend/src/app/messages/messages.component.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { Http } from '@angular/http';
```
Comment (CorneeSean) [inline frontend/src/app/messages/messages.component.ts]: Please migrate to new HttpClient

File: frontend/src/app/messages/messages.component.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  templateUrl: './messages.component.html',
+})
+export class MessagesComponent {
+  messages$: Observable<Message[]>;
+  showMessages = true;
+
+  constructor(private http: Http) {
+    this.messages$ = this.http.get(
+      `${environment.apiRoot}/messages/`,
+      { withCredentials: true }
```
Comment (CorneeSean) [inline frontend/src/app/messages/messages.component.ts]: I guess we don't need "withCredentials" here now with the new interceptor.

File: frontend/src/app/messages/messages.component.ts (hunk 2 above)
Comment (CorneeSean) [reply frontend/src/app/messages/messages.component.ts]: from @angular/common/http

Comment (magul) [reply frontend/src/app/messages/messages.component.ts]: good catch! I will update it to use new http infrastructure!

**Comment by magul** [comment]
@CorneeSean done!

**Pull request merged**